src/components/sections/two-columns.json

```diff
@@ -0,0 +1,75 @@
+{
+  "collectionName": "components_sections_two_columns",
+  "info": {
+    "displayName": "twoColumns",
+    "description": ""
+  },
+  "options": {},
+  "attributes": {
+    "name": {
+      "type": "string",
+      "required": true,
+      "maxLength": 256
+    },
+    "isBottomWaved": {
```

Comment:
Remark:

We don't have waves at the bottom of a section, so this flag is invalid.

Reply:
Please discuss it with BA Iryna Grubman
As I see this field here https://kb.epam.com/display/EPMGCIP/Columns+2